### ISSUE TITLE
Icons missing in forced high contrast mode

### ISSUE THREAD

**Issue by mayank99** [mdn/yari#5944]
### Summary

When using [Windows forced high contrast mode](https://support.microsoft.com/en-us/windows/change-color-contrast-in-windows-fedc744c-90ac-69df-aed5-c8a90125e696#WindowsVersion=Windows_11), some of the icons are missing. In some cases, this is not bad as there is a label which is sufficient, but in other cases (e.g. search input), this makes it hard to understand the purpose of that part of the UI.

### URL

https://developer.mozilla.org and all urls under it

### Reproduction steps

1. Open any MDN webpage
2. Enable forced high contrast mode
   - On Windows 11, this is under Settings > Accessibility > Contrast themes.
   - On Windows 10, this is under Settings > Ease of Access > High contrast.
   - On macOS, this can be emulated in chrome/edge devtools using the "Emulate forced-colors: active" option.

### Expected behavior

Icons should appear, same as in regular theme
![](https://user-images.githubusercontent.com/9084735/162631212-a0ae5f87-2858-4dff-a0e3-c1cbd5a760e7.png)

### Actual behavior

Icons dont appear in high contrast themes
![](https://user-images.githubusercontent.com/9084735/162631150-881e6775-e57b-4379-b45a-0de6cb94ef5e.png)

### Device

Laptop

### Browser

Chrome

### Browser version

Stable

### Operating system

Windows

### Screenshot

<img width="606" alt="image" src="https://user-images.githubusercontent.com/9084735/162631366-2a262c64-168d-40ab-b0f4-387f4d85a039.png">

### Anything else?

I noticed that the icons are using `mask` with `background-color`, so a potential fix would be to add something like this to all the icons:

```css
@media (forced-colors: active) {
  background-color: CanvasText;
}
```

I can potentially create a PR if this is acceptable.

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/yari/discussions).

**Comment by schalkneethling** [comment]
> I can potentially create a PR if this is acceptable.

That would be great, thank you for offering @mayank99!
